integration/setup/config.go

```diff
@@ -63,6 +65,10 @@ func NewConfig() (Config, error) {
 	{
 		c := k8sclient.ClientsConfig{
 			Logger: logger,
+			SchemeBuilder: k8sclient.SchemeBuilder{
+				corev1alpha1.AddToScheme,
+				providerv1alpha1.AddToScheme,
```

Comment:
Don't we need `infrastructurev1alpha2.AddToScheme` here?

Reply:
No because this is legacy. All we need from the CAPI types is to create the new AWSCluster CRD so the `ipam` resource does not error.

`err := config.CPCRDClient.EnsureCreated(ctx, infrastructurev1alpha2.NewAWSClusterCRD(), b)`